torch/testing/_internal/optests/make_fx.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.testing._internal.common_methods_invocations import wrapper_set_seed
+from .common import TestFrameworkError
+
+
+def make_fx_check(func, args, kwargs, tracing_mode, assertion_fn=torch.testing._comparison.assert_close):
```

Comment:
You probably should mention that this will destructively update args

Reply:
I didn't realize, this was a copy-paste. I'll change it to not actually update args, because we re-use them.